lib/licensed/cli.rb

```diff
@@ -46,13 +46,15 @@ def list
       run Licensed::Commands::List.new(config: config), sources: options[:sources], reporter: options[:format], licenses: options[:licenses]
     end
 
-    desc "notices", "Generate a NOTICE file from cached records"
+    desc "notices", "Generate a NOTICE file with dependency data"
     method_option :config, aliases: "-c", type: :string,
       desc: "Path to licensed configuration file"
     method_option :sources, aliases: "-s", type: :array,
       desc: "Individual source(s) to evaluate.  Must also be enabled via configuration."
+    method_option :computed, aliases: "-l", type: :boolean,
+      desc: "Whether to generate a NOTICE file using computed data or cached records"
```

Comment:
Should this option be added to [the `licensed notices` command documentation](https://github.com/github/licensed/blob/c103b2ff8108c05f5bc79e1eae3527dfde20293e/docs/commands/notices.md)?

Reply:
🤦 yep, I made those changes but looks like I never pushed them up to GitHub.  Thanks for catching that!